### DISCUSSION TITLE
How to use the 'datetime local' component, where value is a second timestamp instead of a string of 'YYYY-MM ddTHH: mm'?

### DISCUSSION THREAD

**Discussion by zhanjunjie2019** [formkit/formkit#759] [General]
How to use the 'datetime local' component, where value is a second timestamp instead of a string of 'YYYY-MM ddTHH: mm'?

**Comment by andrew-boyd** [comment]
The `datetime-local` input is the browser native `datetime-local` input — so you need to convert your value into the expected format: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local
